test/e2e/benchmark/throughput.go

```diff
@@ -24,17 +24,17 @@ func getBigBlockManifest() (Manifest, error) {
 		Validators: 2,
 		TxClients:  2,
 		ValidatorResource: testnet.Resources{
-			MemoryRequest: "12Gi",
-			MemoryLimit:   "12Gi",
-			CPU:           "8",
-			Volume:        "20Gi",
-		},
-		TxClientsResource: testnet.Resources{
-			MemoryRequest: "1Gi",
-			MemoryLimit:   "3Gi",
-			CPU:           "2",
-			Volume:        "1Gi",
-		},
+		MemoryRequest: resource.MustParse("12Gi"),
+		MemoryLimit:   resource.MustParse("12Gi"),
+		CPU:           resource.MustParse("8"),
+		Volume:        resource.MustParse("20Gi"),
+	    },
+	    TxClientsResource: testnet.Resources{
+		MemoryRequest: resource.MustParse("1Gi"),
+		MemoryLimit:   resource.MustParse("3Gi"),
+		CPU:           resource.MustParse("2"),
+		Volume:        resource.MustParse("1Gi"),
+        },
```

Comment:
the indentation looks off and golangci-lint is complaining: https://github.com/celestiaorg/celestia-app/actions/runs/10950431497/job/30405641643?pr=3866#step:5:32

Reply:
once you fix lint I'll reapprove @staheri14